### ISSUE TITLE
pyload api wrong format

### ISSUE THREAD

**Issue by kasperj** [Flexget/Flexget#1145]
```
> 2016-05-06 20:49 DEBUG    utils.requests serienjk_de_local Fetching http://192.168.22.11:8000/api/login
> 2016-05-06 20:49 DEBUG    pyload        serienjk_de_local Parsing url ''
> 2016-05-06 20:49 DEBUG    utils.requests serienjk_de_local Fetching http://192.168.22.11:8000/api/parseURLs
> 2016-05-06 20:49 DEBUG    pyload        serienjk_de_local Add 1 urls to pyLoad
> 2016-05-06 20:49 DEBUG    utils.requests serienjk_de_local Fetching http://192.168.22.11:8000/api/addPackage
> 2016-05-06 20:49 DEBUG    entry         serienjk_de_local Marking entry '[DEUTSCH] The.Flash.S02E09.Trio.Infernal.GERMAN.DUBBED.DL.1080p.WebHD.x264-TVP' as failed
> 2016-05-06 20:49 ERROR    entry         serienjk_de_local Failed [DEUTSCH] The.Flash.S02E09.Trio.Infernal.GERMAN.DUBBED.DL.1080p.WebHD.x264-TVP (500 Server Error: Internal Server Error for url: http://192.168.22.11:8000/api/addPackage)
> 2016-05-06 20:49 DEBUG    failed        serienjk_de_local Marking [DEUTSCH] The.Flash.S02E09.Trio.Infernal.GERMAN.DUBBED.DL.1080p.WebHD.x264-TVP in failed list. Has failed 1 times.
> 2016-05-06 20:49 DEBUG    backlog       serienjk_de_local Updating expiry time for [DEUTSCH] The.Flash.S02E09.Trio.Infernal.GERMAN.DUBBED.DL.1080p.WebHD.x264-TVP
> 2016-05-06 20:49 INFO     task          serienjk_de_local Plugin pyload has requested task to be ran again after execution has completed.
> 2016-05-06 20:49 VERBOSE  task          serienjk_de_local FAILED: `[DEUTSCH] The.Flash.S02E09.Trio.Infernal.GERMAN.DUBBED.DL.1080p.WebHD.x264-TVP` by pyload plugin because 500 Server Error: Internal Server Error for url: http://192.168.22.11:8000/api/addPackage
> 2016-05-06 20:49 DEBUG    pyload        serienjk_de_local Parsing url ''
> 2016-05-06 20:49 DEBUG    utils.requests serienjk_de_local Fetching http://192.168.22.11:8000/api/parseURLs
> 2016-05-06 20:49 DEBUG    pyload        serienjk_de_local Add 1 urls to pyLoad
> 2016-05-06 20:49 DEBUG    utils.requests serienjk_de_local Fetching http://192.168.22.11:8000/api/addPackage
> 2016-05-06 20:49 DEBUG    entry         serienjk_de_local Marking entry '[DEUTSCH] Unforgettable.S04E13.Die.geheime.Armee.German.DD51.Dubbed.DL.1080p.Web-DL.h264-GDR' as failed
> 2016-05-06 20:49 ERROR    entry         serienjk_de_local Failed [DEUTSCH] Unforgettable.S04E13.Die.geheime.Armee.German.DD51.Dubbed.DL.1080p.Web-DL.h264-GDR (500 Server Error: Internal Server Error for url: http://192.168.22.11:8000/api/addPackage)
> 2016-05-06 20:49 DEBUG    failed        serienjk_de_local Marking [DEUTSCH] Unforgettable.S04E13.Die.geheime.Armee.German.DD51.Dubbed.DL.1080p.Web-DL.h264-GDR in failed list. Has failed 1 times.
> 2016-05-06 20:49 DEBUG    backlog       serienjk_de_local Updating expiry time for [DEUTSCH] Unforgettable.S04E13.Die.geheime.Armee.German.DD51.Dubbed.DL.1080p.Web-DL.h264-GDR
> 2016-05-06 20:49 DEBUG    task          serienjk_de_local Plugin pyload has requested task to be ran again after execution has completed.
> 2016-05-06 20:49 VERBOSE  task          serienjk_de_local FAILED: `[DEUTSCH] Unforgettable.S04E13.Die.geheime.Armee.German.DD51.Dubbed.DL.1080p.Web-DL.h264-GDR` by pyload plugin because 500 Server Error: Internal Server Error for url: http://192.168.22.11:8000/api/addPackage
```
Error 500 is typical for a format error for the pyload api

https://github.com/pyload/pyload/blob/stable/module/Api.py

I'm not common to python, could someone plz send me a sample output, so i can compare the formats?

**Comment by jbman11** [comment]
I am happy to help. Please let me know what commands I need to do to get you the output you need

**Comment by kasperj** [comment]
A manual api call adding a package works fine, so it is about the format of the flexget query, but because i don't know python very well I have no idea how to print that query to log or somewhere else

**Comment by Leonderi** [comment]
Is there someone who can fix this? I think i have the same problem..

http://pastebin.com/bsjp0wQH

**Comment by cvium** [comment]
This may have been fixed by #1391. Someone please check.

**Comment by cvium** [comment]
I'm just going to assume it's been fixed. Let me know if it happens again.
